src/index.html

```diff
@@ -1,14 +1,43 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+    href="https://fonts.gstatic.com" crossorigin="true">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#" class="moyo">
+        <img src="./images/logo.png" alt="moyo">
+      </a>
+      <nav>
+        <ul class="contener">
+          <li><a href="#" class="is-active">Apple</a></li>
+          <li><a href="#" class="is-active">Samsung</a></li>
+          <li><a href="#" class="is-active">Smartphones</a></li>
+          <li>
+            <a href="#"
+            class="is-active"
+            data-qa="hover">Laptops & Computers</a>
```

Comment:
Fix code style everywhere
```suggestion
            <a
              href="#"
              class="nav__link is-active"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```